lib/src/SearchBar.js

```diff
@@ -50,7 +53,7 @@ export default class SearchBar extends Component {
           styles.center,
           container(this.props),
           ifIPhoneXHeader(noExtraMargin),
-          _shadowStyle(shadowColor)
+          _shadowStyle(shadowColor, shadowStyle)
```

Comment:
What we need is: 
```jsx
{ shadowStyle ||  _shadowStyle(shadowColor) }
```

Can you change it something like that? Therefore, everyone can simply integrate their own shadow style easily.